src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -0,0 +1,36 @@
+package dk.cngroup.lentils.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+
+@Service
+public class ImageService {
+
+    public static final String IMAGE_DIRECTORY = "\\images\\";
+
+    public void readAndSaveImageFile(final MultipartFile file) {
+        if (!file.isEmpty()) {
+            String fileName = file.getOriginalFilename();
+            String pathToSave = getDirectory() + fileName;
+            String fileExtension = fileName.substring(fileName.lastIndexOf(".") + 1);
+            File dest = new File(pathToSave);
+            try {
+                file.transferTo(dest);
+            } catch (IOException e) {
+                System.err.println("Picture file cannot be saved. " + e.getMessage());
```

Comment:
Toto taky neni dobre. Tady bychom meli vyhodit vyjimku tak, abychom dali vedet zpatky klientovi, ze se nahrani nepodarilo z nejakeho duvodu.

Reply:
Ale zas pri editaci treba napovedy, kdyz chces zmenit jen text a nezadas soubor, tak nechces aby to hlasilo chybu.
nebo jak to bude v UI?